amida-tech/orange-ignite
# Feature/orange 75 external registration

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do? Adds external registration (currently Facebook)

#### Related JIRA tickets: ORANGE-75

#### How should this be manually tested?

This should be a tested with the corresponding back end also in PR. I added test users zhnbglkebg_1519926338@tfbnw.net (pw: amida1) and wgullicvff_1519926847@tfbnw.net (pw: amida2).

1) Specify EXTERNAL_AUTH = 'facebook' in the config file.
2) Launch the app. Click on Sign Up. Facebook should be launched to login. If you login your user will be registered and you will be brought to the Main Screen.
3) For a registered user you can click on Sign In button and you can login using Facebook.
4) Sign In with a non registered user or registering an already registered user should give an error.

#### Background/Context NA

#### Screenshots (if appropriate): NA
